src/js/profile/wearable/widget/wearable/Datetimepicker.js

```diff
@@ -135,7 +135,7 @@
 					direction = -1;
 					// Everything else than minus / plus button should be ignored
 				} else if (!buttonClassList.contains(classes.uiDatetimeWheelPlus)) {
-					return null;
+					return;
```

Comment:
Remove @return {null} from js doc for this method

Reply:
Right. Thanks.